lib/utils/dataStructureUtils.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+function createMapFromObject(obj) {
+    const map = new Map();
+    for (const key in obj) {
+        map.set(key, obj[key]);
+    }
+    return map;
+}
+exports.createMapFromObject = createMapFromObject;
+function isArrayOfString(x) {
+    return (Array.isArray(x) &&
+        x.every(item => Boolean(item && typeof item === "string")));
+}
+exports.isArrayOfString = isArrayOfString;
+function isArrayOfObject(x) {
+    return (Array.isArray(x) &&
+        x.every(item => Boolean(item && typeof item === "object")));
```

Comment:
I guess `item && ` check is redundant. `every` method should give an item all the time. Doesn't it?

Reply:
Yes, it passes the tests when the `item &&` check is removed.